test_data.rb

```diff
@@ -448,4 +447,35 @@ def test_sncf_virtual_station
       assert_equal "t", station["sncf_is_enabled"], "Virtual station #{station["name"]} (#{station["id"]}) should be enabled for SNCF"
     end
   end
+
+  AIRPORT_TRANSLATIONS = {
+    :fr => 'aeroport',
+    :en => 'airport',
+    :de => 'flughafen',
+    :es => 'aeropuerto',
+    :it => 'aeroporto',
+    :nl => 'luchthaven',
+    :da => 'lufthavn',
+    :ru => 'аэропорт'
+  }
+
+  def test_airports_comments_and_children
+    STATIONS.select do |row|
+      row['is_airport'] == 't'
+    end.each do |row|
+
+      assert (AIRPORT_TRANSLATIONS.none? do |locale, translation|
+        comment = slugify(row["comments:#{locale}"] || '')
+        comment =~ /#{translation}/i
+      end), "Redundant comment for #{row['name']} station"
```

Comment:
If we check that, it'll fail, and I prefer not to touch stations names for the moment.

Ok I'll use a different message, thanks